prophy/generators.py

```diff
@@ -104,6 +173,17 @@ def check(cls, value):
         self._int_to_name = int_to_name
         self._check = classmethod(check)
 
+    def __eq__(cls, other):
+        if not isinstance(other, cls.__class__):
+            return NotImplemented
+        return cls._enumerators == other._enumerators
+
+    def __ne__(cls, other):
+        are_equal = cls.__class__.__eq__(cls, other)
+        if are_equal is NotImplemented:
+            return NotImplemented
+        return not are_equal
```

Comment:
Altough this  implementation of `__eq__` differs from `_generator_base` the `__ne__` is the same and can be removed.